src/Microsoft.Framework.Caching.Redis/CachingServiceExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Caching.Distributed;
+using Microsoft.Framework.Caching.Redis;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class CachingServicesExtensions
+    {
+        public static IServiceCollection AddRedisCache(this IServiceCollection collection)
+        {
+            collection.AddOptions();
+            collection.TryAdd(ServiceDescriptor.Transient<IDistributedCache, RedisCache>());
```

Comment:
curious if this should be a Singleton?...the current Redis cache implementation, which depends on StackExchange Redis client, talks about caching the ConnectionMultiplexer once its created...was just reading this : https://github.com/StackExchange/StackExchange.Redis/blob/master/Docs/Basics.md